Oops! Assertion failed

The following assertion was not satisfied
  assert "hello" == "foo bar"\
      and 3 == 3\
      and 1 > 1 or 1>1 or 3>3 or 4>+4, error()

Assertion values
  assert "hello" == "foo bar"\
                 ^ is = False
      and 3 == 3\
      and 1 > 1 or 1>1 or 3>3 or 4>+4, error()
  assert "hello" == "foo bar"\
      and 3 == 3\
            ^ is = True
      and 1 > 1 or 1>1 or 3>3 or 4>+4, error()
  assert "hello" == "foo bar"\
      and 3 == 3\
      ^ is = False
      and 1 > 1 or 1>1 or 3>3 or 4>+4, error()
  assert "hello" == "foo bar"\
      and 3 == 3\
      and 1 > 1 or 1>1 or 3>3 or 4>+4, error()
            ^ is = False
  assert "hello" == "foo bar"\
      and 3 == 3\
      and 1 > 1 or 1>1 or 3>3 or 4>+4, error()
      ^ is = False
  assert "hello" == "foo bar"\
      and 3 == 3\
      and 1 > 1 or 1>1 or 3>3 or 4>+4, error()
                    ^ is = False
  assert "hello" == "foo bar"\
      and 3 == 3\
      and 1 > 1 or 1>1 or 3>3 or 4>+4, error()
                ^ is = False
  assert "hello" == "foo bar"\
      and 3 == 3\
      and 1 > 1 or 1>1 or 3>3 or 4>+4, error()
                           ^ is = False
  assert "hello" == "foo bar"\
      and 3 == 3\
      and 1 > 1 or 1>1 or 3>3 or 4>+4, error()
                       ^ is = False
  assert "hello" == "foo bar"\
      and 3 == 3\
      and 1 > 1 or 1>1 or 3>3 or 4>+4, error()
                                   ^ is = 4
  assert "hello" == "foo bar"\
      and 3 == 3\
      and 1 > 1 or 1>1 or 3>3 or 4>+4, error()
                                  ^ is = False
  assert "hello" == "foo bar"\
      and 3 == 3\
      and 1 > 1 or 1>1 or 3>3 or 4>+4, error()
                              ^ is = False
  assert "hello" == "foo bar"\
  ^ is False
      and 3 == 3\
      and 1 > 1 or 1>1 or 3>3 or 4>+4, error()

Where
  File 'tests/tests.py', line 47 in 'run'

39|                      test.note(e.exception)
40|                      with this() as that:
41|                          assert that(snapshot(e.exception, "multiline-implicit", encoder=str)), error()
42|  
43|                  with suite.test("explicit") as test:
44|                      with raises(AssertionError) as e:
45|                          assert "hello" == "foo bar"\
46|                              and 3 == 3\
47|>                             and 1 > 1 or 1>1 or 3>3 or 4>+4, error()
48|                      test.note(e.exception)
49|                      with this() as that:
50|                          assert that(snapshot(e.exception, "multiline-explicit", encoder=str)), error()
